apps/admin-portal/src/components/claims/add/add-external-claims.tsx

```diff
@@ -47,7 +47,7 @@ export const AddExternalClaims = (props: AddExternalClaimsPropsInterface): React
         }).catch(error => {
             dispatch(addAlert(
                 {
-                    description: error?.description,
+                    description: error?.description || "There was an error while fetching local claims",
```

Comment:
We are getting this error for multiple adding or sing adding? If it's single error message needs to fix.

Reply:
This error is shown when you fail to fetch all the local claims. Since when adding an external claim, you are supposed to map it to a local claim, we have to load all the local claims.